.gitignore

```diff
@@ -13,4 +13,4 @@ _themes.VS.Modern/
 
 # Documentation build
 /docs/vcppdocs
```

Comment:
What is this change about? Can you revert this one?

Reply:
Thank you for your reminding.
Didn't notice a blank line at the end, deleted.